instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py

```diff
@@ -38,7 +38,11 @@
 
 major_version = elasticsearch.VERSION[0]
 
+NodeAPIResponse, ApiResponseMeta, HttpHeaders = None, None, None
+
 if major_version == 8:
+    from elastic_transport._node import NodeApiResponse
```

Comment:
Let's use that instead. I made a mistake when reviewing a pull request that exposed `NodeApiResponse`, it should be `TransportApiResponse` instead. That way in the future you will avoid the private import.

```suggestion
    from elastic_transport._transport import TransportApiResponse
```

Reply:
With that means i should also patch at the Transport.perform_request level instead of the Urllib3 backend right?

Reply:
Yes, which will cover more backends.

Reply:
Ok, will revise once tests pass with 8.12

Reply:
Transport.perform_request is the method we are wrapping for our instrumentation so mocking one level below in tests seems right to avoid testing our own mocks